### PR TITLE
(WIP) Supermarket options

### PR DESCRIPTION
**Pull request by brewn**

### CODE REVIEW

File: chef_master/source/config_rb_supermarket.rst [hunk 1]
```diff
@@ -476,89 +523,41 @@ StatsD
...
 
 ``default['supermarket']['chef_domain']``
-   Default value: ``'chef.io'``.
+   The root domain that is used by all Chef URLs. Most of the settings in this section rely upon this setting. Default value: ``'chef.io'``.
 
 ``default['supermarket']['chef_identity_url']``
-   Default value: ``"#{node['supermarket']['chef_server_url']}/id"``.
+   The URL that is used to interact with Chef Identity on the Chef server. Default value: ``"#{node['supermarket']['chef_server_url']}/id"``.
 
 ``default['supermarket']['chef_manage_url']``
```
Comment (brewn) [inline chef_master/source/config_rb_supermarket.rst]: I can probably get rid of this one, right?

File: chef_master/source/config_rb_supermarket.rst [hunk 2]
```diff
@@ -5,84 +5,75 @@ supermarket.rb Settings
...
-.. note:: When changes are made to the chef-server.rb file the Chef server must be reconfigured by running the ``supermarket-ctl reconfigure`` command.
+.. note:: You must run ``supermarket-ctl reconfigure`` to apply any changes made in the ``supermarket.rb`` file.
 
 General
 -----------------------------------------------------
 This configuration file has the following general settings:
 
 ``default['enterprise']['name']``
-   Default value: ``'supermarket'``.
+   The Chef Automate enterprise name. Default value: ``'supermarket'``.
```
Comment (robbkidd) [inline chef_master/source/config_rb_supermarket.rst]: This `enterprise` bit isn't related to Automate. This is an attribute used by the `enterprise` cookbook.

**Review by robbkidd — CHANGES_REQUESTED** [review]
All looks good save for the confusion around "enterprise."

File: chef_master/source/config_rb_supermarket.rst (hunk 2 above)
Comment (brewn) [reply chef_master/source/config_rb_supermarket.rst]: heh yeah, in retrospect that makes sense given the default value.

**Pull request merged**